gremlin-driver/src/main/java/org/apache/tinkerpop/gremlin/driver/handler/HttpGremlinRequestEncoder.java

```diff
@@ -68,14 +71,15 @@ protected void encode(final ChannelHandlerContext channelHandlerContext, final R
                     requestMessage));
         }
 
+        InetSocketAddress remoteAddress = getRemoteAddress(channelHandlerContext.channel());
```

Comment:
Nit: add `final`. Please check over other parts that may need similar update.

Reply:
Added a couple finals